api/v1/constructors_test.go

```diff
@@ -1073,6 +1073,44 @@ var _ = Describe("Constructor utils for kind", func() {
 			})
 		})
 	})
+
+	Describe("Test parseCertificateInfo filtering certificates", func() {
+		Context("When non empty CertificateInfo is given with openldap/docker_registry/ssl cert included", func() {
+			It("Populates the spec with CertificateInfo without the openldap/docker_registry/ssl cert", func() {
+				spec := &SystemSpec{}
+				certificates := []certificates.Certificate{
+					{
+						Type:      "T1",
+						Signature: "hash1",
+					},
+					{
+						Type:      "openldap",
+						Signature: "hash2",
+					},
+					{
+						Type:      "docker_registry",
+						Signature: "hash3",
+					},
+					{
```

Comment:
Need to add `Type: openstack_ca` as well to this list and modify certInfo accordingly.

Reply:
Done.